src/client/src/ui/spotTile/components/notifications/TileBooking.tsx

```diff
@@ -13,22 +13,21 @@ const TileBookingStyle = styled('div')`
 `
 
 const BookingPill = styled(Flex)`
-  background-color: ${({ theme: { palette } }) => palette.accentPrimary.normal};
+  background-color: ${({ theme }) => theme.primary.base};
   padding: 0.625rem 0.875rem;
   border-radius: 17px;
 
-  box-shadow: 0 0 30px ${({ theme: { palette } }) => palette.accentPrimary.normal},
-    0 0 0.625rem ${({ theme: { palette } }) => palette.accentPrimary.dark};
+  box-shadow: 0 0 30px ${({ theme }) => theme.primary.base}, 0 0 0.625rem ${({ theme }) => theme.primary.light};
```

Comment:
This is an interesting example — your use case for the shadow implies to me the opposite result.

Because we want a crisp visual around the entity we'd choose a pixel value that'll remain consistent — and because we have a large glow, we'd choose a relative value because that would scale proportionally with all other elements.

I'd write `box-shadow: 0 0 2rem ${({ theme }) => theme.primary.base}, 0 0 8px ${({ theme }) => theme.primary.light};`

Unless they both should scale, and then I'd write `box-shadow: 0 0 2rem ${({ theme }) => theme.primary.base}, 0 0 .5rem ${({ theme }) => theme.primary.light};`